### PR TITLE
Add GA CI workflow

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by ADmad** [comment]
@othercorey Any idea what min. version of twig-view plugin do we need for the `prefer-lowest` build to pass?

**Comment by othercorey** [comment]
> @othercorey Any idea what min. version of twig-view plugin do we need for the `prefer-lowest` build to pass?

We should enforce >= 1.0.2.

**Comment by othercorey** [comment]
> cakephp/plugin-installer (1.0.0)

might be the problem?

**Comment by ADmad** [comment]
Hah, the failure is due to combination of composer v2 (default used by `shivammathur/setup-php`) and plugin-installer v1 (due `--prefer-lowest`).

File: composer.json [hunk 1]
```diff
@@ -19,13 +19,14 @@
     },
     "require": {
         "php": ">=7.2",
-        "cakephp/cakephp": "^4.0",
-        "cakephp/twig-view": "^1.0"
+        "cakephp/cakephp": "^4.1",
```
Comment (ADmad) [inline composer.json:22]: This was required to fix following failure with `--prefer-lowest`:

```
1) Bake\Test\TestCase\Command\ModelCommandTest::testApplyAssociations
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
-    0 => 'Users'
-    1 => 'TodoTasks'
-    2 => 'TodoLabels'
+    0 => 'users'
+    1 => 'todotasks'
+    2 => 'todolabels'
 )
```

**Comment by othercorey** [comment]
Can we disable the double push/pr build?

**Comment by ADmad** [comment]
> Can we disable the double push/pr build?

Done

**Pull request merged**